docs/rest_api.md

```diff
@@ -2,3 +2,266 @@
+```
+
+To perform a blocking invocation on an action, send a HTTP request with a method `POST` and body containing the input parameter `name` use the following:
+```bash
+curl -u $AUTH -d '{"name":"John"}' -X POST -H "Content-Type: application/json" "https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/actions/hello?blocking=true"
+```
+You get the following response:
+```json
+{
+  "duration": 2,
+  "name": "hello",
+  "subject": "john@example.com_dev",
+  "activationId": "c7bb1339cb4f40e3a6ccead6c99f804e",
+  "publish": false,
+  "annotations": [{
+    "key": "limits",
+    "value": {
+      "timeout": 60000,
+      "memory": 256,
+      "logs": 10
+    }
+  }, {
+    "key": "path",
+    "value": "john@example.com_dev/hello"
+  }],
+  "version": "0.0.1",
+  "response": {
+    "result": {
+      "payload": "Hello John"
+    },
+    "success": true,
+    "status": "success"
+  },
+  "end": 1493327653769,
+  "logs": [],
+  "start": 1493327653767,
+  "namespace": "john@example.com_dev"
+}
+```
+If you just want to get the `response.result` run the command again with the query parameter `result=true`
+```bash
+curl -u $AUTH -d '{"name":"John"}' -X POST -H "Content-Type: application/json" "https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/actions/hello?blocking=true&result=true"
+```
+You get the following response:
+```json
+{
+  "payload": "hello John"
+}
+```
+
+## Annotations and Web Actions
+
+To create an action as a web action, you need to add an [annotation](annotations.md) of `web-export=true` for web actions since they are public you want any default parameter to by final by using the annotation `final=true`
+
+Run the curl command providing the complete list of annotations to set on the action
+```bash
+curl -u $AUTH -X PUT -H "Content-Type: application/json" https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/actions/hello?overwrite=true -d '{"namespace":"_","name":"hello","exec":{"kind":"nodejs:6","code":"function main(params) { return {payload:\"Hello \"+params.name}}"},"annotations":[{"key":"web-export","value":true},{"key":"raw-http","value":false},{"key":"final","value":true}]}'
+```
+You can now invoke this action as a public url with no OpenWhisk authorization. Try invoking using the web action public URL including an optional extension such as `.json` or `http` for example at the end of the URL.
+```bash
+curl https://openwhisk.ng.bluemix.net/api/v1/web/john@example.com_dev/default/hello.json?name=John
+```
+```json
+{
+  "payload": "Hello John"
+}
+```
+
+## Sequences
+
+To create an action sequence, you need to create it by providing the name of the actions that compose the sequence in the desired order, so the output from the first action is passed as input to the next action.
```

Comment:
should clarify that the names of the actions in the component list must be fully qualified.